### PR TITLE
Type the global "jQuery" variable

### PR DESCRIPTION
**Pull request by anandthakker**
Type the global "jQuery" variable (previously "any") and fixup uses of methods we have already removed

### CODE REVIEW

File: src/services/aria.ts [hunk 1]
```diff
@@ -78,7 +82,7 @@ class Aria {
         .replace(/ +(?= )/g, '')
         .trim();
       if (this.controller.containerHasFocus()) {
-        this.jQ.empty().text(this.msg);
+        this.domFragment.oneElement().innerText = this.msg;
```
Comment (jwmerrill) [inline src/services/aria.ts:85]: I think we might want to prefer setting `textContent` over setting `innerText`. I think the result is really similar, but it sounds like setting `innerText` can do weird things in old IE

Ref: https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innertext

File: src/shared_types.d.ts [hunk 2]
```diff
@@ -106,3 +107,7 @@ interface $ {
   length: number;
   [index: number]: HTMLElement; // TODO - this can be undefined. Either fix uses or wait until removing jquery
 }
+
+interface JQueryGlobalObject {
+  contains(c: Element, target: Element): boolean;
+}
```
Comment (jwmerrill) [inline src/shared_types.d.ts:113]: Seems like maybe this isn't referenced anymore.

File: src/utils.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ const L: L = -1;
 const R: R = 1;
 type Direction = L | R;
 
-var jQuery = (window as any).jQuery;
+var jQuery: $ = (window as any).jQuery;
```
Comment (jwmerrill) [inline src/utils.ts:12]: Where do we set the `$` variable? Wondering if maybe we should avoid binding a `jQuery` variable at all. Is it still referenced anywhere else besides line 16 of this file?

**Review by jwmerrill — COMMENTED** [review]
Good catch on this one

**Comment by anandthakker** [comment]
Amended the commit to address review notes, and rebased

**Review by jwmerrill — APPROVED** [review]

**Pull request merged**